plugin/src/main/resources/io/jenkins/plugins/casc/assets/prism.css

```diff
@@ -0,0 +1,125 @@
+/* PrismJS 1.16.0
+https://prismjs.com/download.html#themes=prism-tomorrow&languages=yaml */
+/**
+ * prism.js tomorrow night eighties for JavaScript, CoffeeScript, CSS and HTML
```

Comment:
I think Okaidia would be a better choice

Reply:
feel free to mess around with it 😉 
This one was easy to setup but I'm not attached to it at all

Reply:
Ah you meant the theme yeah sure, pushed
![image](https://user-images.githubusercontent.com/21194782/56345482-bc71e100-61b7-11e9-8d02-a0fa1d3e7c1b.png)